### ISSUE TITLE
False positive on duplicate parameter checks

### ISSUE THREAD

**Issue by vStone** [rodjek/puppet-lint#145]
``` puppet

node test {
  apache::vhost {'my_vhost_80':
    serveralias => ['alias1', 'alias2',],
    mods        => {
      'rewrite'   => [
        { rewrite_rule => '^/google/(.*) https://google.com' },
        { rewrite_rule => '^/vstone/(.*) http://vstone.eu', },
      ],
    },
  }
}
```
